sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -62,19 +68,28 @@ import {
   WILComponentOptions,
 } from "./education-program-offering-validation.models";
 import { EducationProgramOfferingValidationService } from "./education-program-offering-validation.service";
+import { InstitutionLocationService } from "../institution-location/institution-location.service";
 import * as os from "os";
 import { LoggerService, InjectLogger } from "@sims/utilities/logger";
+import {
+  InstitutionAddsPendingOfferingNotification,
+  NotificationActionsService,
+} from "@sims/services";
 @Injectable()
 export class EducationProgramOfferingService extends RecordDataModelService<EducationProgramOffering> {
   constructor(
     private readonly dataSource: DataSource,
     private readonly offeringValidationService: EducationProgramOfferingValidationService,
+    private readonly notificationActionsService: NotificationActionsService,
+    private readonly institutionLocationService: InstitutionLocationService,
+    private readonly entityManager: EntityManager,
```

Comment:
To this day we never follow a similar approach. Is there a benefit to doing it?

Reply:
Hmm. I don't know of any specific advantage. Just that the framework would inject an object into the service and I can use it. Will change it.